protocol/avdl/keybase1/notify_fs.avdl

```diff
@@ -6,5 +6,9 @@ protocol NotifyFS {
   @notify("")
   void FSActivity(FSNotification notification);
 
+  void FSSyncActivity(FSPathSyncStatus status) oneway;
```

Comment:
avdl method names should be lowercase?

also if the protocol is fs... then method name doesn't need to start with fs